tests/foreman/api/test_reporttemplates.py

```diff
@@ -406,3 +423,68 @@ def test_negative_nonauthor_of_report_cant_download_it(self):
 
         :CaseImportance: Medium
         """
+
+    @tier3
+    def test_positive_generate_entitlements_report(self):
+        """Generate a report using the Entitlements template.
+
+        :id: 722e8802-367b-4399-bcaa-949daab26632
+
+        :setup: Installed Satellite with Organization, Activation key,
+                Content View, Content Host, and Subscriptions.
+
+        :steps:
+
+            1. Get /api/report_templates/115-Entitlements/generate/id/report_format
+
+        :expectedresults: Report is generated showing all necessary information for entitlements.
+
+        :CaseImportance: High
+        """
+        org = entities.Organization().create()
+        with manifests.clone() as manifest:
+            upload_manifest(org.id, manifest.content)
+        rh_repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=org.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
```

Comment:
'rhst7' will sync the 6.6 tools[1] repos , So i would recommend using [2] for the latest on version in custom repo:

[1] https://github.com/SatelliteQE/robottelo/blob/master/robottelo/constants.py#L205
[2] https://github.com/SatelliteQE/robottelo/blob/2599f5d763746f0553759291f315e759a85541e5/tests/foreman/api/test_contentaccess.py#L109

Reply:
@ColeHiggins2 looks some you forgot this one :)

Reply:
@vijay8451, so instead of using fhst7, he should use rhel7 for custom sattools_repo so it's the latest.  Is that correct?

Reply:
@latran `REPOS['rhst7']` will call the below which is related to 6.6 not 6.7 ..
```
   'rhst7': {
        'id': 'rhel-7-server-satellite-tools-6.6-rpms',
        'name': (
            'Red Hat Satellite Tools 6.6 for RHEL 7 Server RPMs x86_64'
        ),
        'version': '6.6',
        'reposet': REPOSET['rhst7'],
        'product': PRDS['rhel'],
        'distro': DISTRO_RHEL7,
        'key': 'rhst',
    }
```
hence to call latest(6.7) tool he requires to use as below using custom repo.
```
from robottelo.config import settings
tool_repo = settings.sattools_repo['rhel7']
```

Reply:
@vijay8451 This isn't truly necessary for these tests, as the host just needs to consume any subscription. In many of our other tests, keeping on the latest is definitely preferred, since agent versions should match satellite. However, since agent isn't being used, this subscription will work fine.
@ColeHiggins2 feel free to correct me if I'm mistaken.

Reply:
@JacobCallahan yeah I do understand that .. the idea was to go with compatible package instead of old ones .....
@ColeHiggins2 
However feel free move further(as it was a suggestion only) if this one is not much necessary in these tests and I am fine with it.